dashboard/app/models/code_review_group.rb

```diff
@@ -11,9 +11,11 @@
 # Indexes
 #
 #  index_code_review_groups_on_section_id  (section_id)
+#  TODO: Add index on follower_id
```

Comment:
Maybe a learning moment for me here -- I saw the NO INDEX query in your description...we need to have indexes on tables for columns that aren't even in the table?

Reply:
Oh oops, thanks for catching this, I meant to put this in the code_review_group_members model...  but I see there's already an index on `follower_id` there.  Hmmm, I'm not really sure why rails thinks there's no index then.